### PR TITLE
Signalen Community Day invitation

### PR DESCRIPTION
**Pull request by AlbaRoza**
-----
[View rendered _posts/2021-03-03-signalen-community-meetup-march-22.md](https://github.com/publiccodenet/blog/blob/New-blogpost/_posts/2021-03-03-signalen-community-meetup-march-22.md)

### CODE REVIEW

**Comment by AlbaRoza** [comment]
Creation of blogpost with the information from the Signalen Community Meetup that will be held the following March 22.

**Review by AlbaRoza — COMMENTED** [review]
Version without the latest line.

File: _posts/2021-03-03-signalen-community-meetup-march-22.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Invitation: Signalen community meetup, March 22nd"
+date: 2021-03-03
+author: Alba Roza
+type: blogpost
+excerpt: New Signalen community day
+categories:
```
Comment (ElenaFdR) [inline _posts/2021-03-03-signalen-community-meetup-march-22.md:7]: Until we switch from categories to tags, each blogpost can only have 1 category.

File: _posts/2021-03-03-signalen-community-meetup-march-22.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - Signalen
+  - Community
+---
+
+# Invitation: Signalen community meetup, March 22nd
+
+![Invitation: Signalen invitation community meetup](/assets/Signalen-banner-EN.png)
+
+A few weeks ago we announced in this blog that Signalen was in incubation with the Foundation for Public Code. Today we would like to invite you to its second community get together, that will be held online the following March 22.
```
Comment (ElenaFdR) [inline _posts/2021-03-03-signalen-community-meetup-march-22.md:16]: Add a hyperlink to the previous blogpost.

**Comment by AlbaRoza** [comment]
Changed the text with the suggested changes by @ElenaFdR

**Review by ElenaFdR — APPROVED** [review]
Totally ready to launch!

**Pull request merged**